sphinx/builders/__init__.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, app):
                                        self.versioning_compare)
         self.srcdir = app.srcdir
         self.confdir = app.confdir
-        self.outdir = app.outdir
+        self.outdir = "build/{:}".format(self.name)
```

Comment:
Just a kludge to support multiple different output dirs. Would need a better solution if I moved forward.